packages/dapp/src/components/ResolveFunds.jsx

```diff
@@ -143,8 +160,21 @@ export const ResolveFunds = ({ invoice, balance, close, tokenData }) => {
               <InputRightElement w="3.5rem">{symbol}</InputRightElement>
             </InputGroup>
           </VStack>
-          <VStack spacing="0.5rem" align="stretch" color="red.500">
-            <Text fontWeight="700">Provider Award</Text>
+          <VStack spacing="0.5rem" align="stretch" color="black">
+            {/* <Text fontWeight="700">Provider Award</Text> */}
+            <Flex w="100%">
+              <Text fontWeight="700" color="black">
+                Provider Award
```

Comment:
We need some consistency whether we choose provider or freelancer. We should refrain from using the two words interchangeably as it may confuse the user a bit.